crates/turborepo-lib/src/shim.rs

```diff
@@ -585,12 +585,25 @@ fn run_correct_turbo(
         try_check_for_updates(&shim_args, &turbo_state.version);
 
         if turbo_state.local_is_self() {
+            // `spawn_local_turbo`` sets the env var, so we can check for it here as an
+            // indication of whether we've been spawned from a global binary
+            let spawned_from_global = env::var(cli::INVOCATION_DIR_ENV_VAR).is_ok();
```

Comment:
cc @chris-olszewski on this one. This looks to be the only way to detect this right now, but happy to change to something better if it exists!

Reply:
I don't believe this updated codepath ever gets hit as it should only happen if the turbo binary doesn't support `--skip-infer`.